domain/src/main/java/starter/project/domain/Doors.java

```diff
@@ -6,11 +6,14 @@
 import lombok.Data;
 import lombok.NoArgsConstructor;
 
+import javax.persistence.Embeddable;
```

Comment:
This import is not being used, it can be removed

Reply:
resolved